### DISCUSSION TITLE
Suggestion for rephrasing some translation strings

### DISCUSSION THREAD

**Discussion by kajaji** [Stellarium/stellarium#1562] [Translations]
In the Stellarium translation file:
#: src/ui_viewDialog.h:3821
msgid "Set the planet and position as it specified in landscape.ini"

The casual users may not know what "landscape.ini" is, so I suggest modifying the phrase to something like:
"Set the planet and position as specified for the selected landscape".

#: src/ui_astroCalcExtraEphemerisDialog.h:193
msgid "Skip draw the unlabelled markers also"

IMHO it needs to be rephrased.

#: src/ui_shortcutsDialog.h:212
msgid "Search action..."

It is not clear that it means "Enter search string here" without looking at it in the context of the keyboard shortcut window.

**Comment by alex-w** [comment]
> In the Stellarium translation file:
> #: src/ui_viewDialog.h:3821
> msgid "Set the planet and position as it specified in landscape.ini"
> 
> The casual users may not know what "landscape.ini" is, so I suggest modifying the phrase to something like:
> "Set the planet and position as specified for the selected landscape".

OK, this is good proposal

> #: src/ui_astroCalcExtraEphemerisDialog.h:193
> msgid "Skip draw the unlabelled markers also"
> 
> IMHO it needs to be rephrased.

Probably yes, but this part of GUI will be refactored in next release (I hope I will done it in 0.21.1)

> #: src/ui_shortcutsDialog.h:212
> msgid "Search action..."
> 
> It is not clear that it means "Enter search string here" without looking at it in the context of the keyboard shortcut window.

Hmm. Have I lost the article in a phrase? @gzotti

**Comment by alex-w** [comment]
Please check the fresh version (development snapshot) of Stellarium:
https://github.com/Stellarium/stellarium-data/releases/tag/weekly-snapshot

**Reply by kajaji** [reply]
Thanks @alex-w , rc2 downloaded and tested.
